.travis.yml

```diff
@@ -0,0 +1,16 @@
+# Config file for automatic testing at travis-ci.org
+
+language: python
+
+python:
+  - "3.4"
+  - "3.3"
+  - "2.7"
+  - "2.6"
+  - "pypy"
+
+# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+install: pip install -r requirements.txt
```

Comment:
What about running `python setup.py install` or `pip install .` instead? This way we'll be able to check missing dependencies in setup.py.